cloud

```diff
@@ -1251,5 +1272,4 @@ function execute() {
                     --log-level="$log_level" \
                     --submitters="${submitters[*]@Q}"
 }
-
-execute "$@"
+execute "$@"
```

Comment:
Could you restore the newline after this line? If we keep adding and removing them, it will be just unnecessary noise in git history.